src/interfaces/reactNativeInterface.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  create(styleHash) {
```

Comment:
Right, this is what is called to prepare the styles so this is always an object.